packages/secure_storage/amplify_secure_storage/example/integration_test/cupertino_test.dart

```diff
@@ -16,18 +17,19 @@ const key2 = 'key_2';
 const value1 = 'value_1';
 const value2 = 'value_2';
 
-// Enabling useDataProtection requires that the app is
-// added to at least one access group.
-final macOSOptions = MacOSSecureStorageOptions(useDataProtection: false);
-
-/// MacOS & iOS app uninstall & re-install tests.
-///
-/// These tests are only relevant for amplify_secure_storage
-/// (not amplify_secure_storage_dart) as they use
-/// NSUserDefaultsAPI via pigeon.
 void main() {
   IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+
+  /// MacOS & iOS app uninstall & re-install tests.
+  ///
+  /// These tests are only relevant for amplify_secure_storage
+  /// (not amplify_secure_storage_dart) as they use
+  /// NSUserDefaultsAPI via pigeon.
```

Comment:
The new tests can be done in Dart too, though, no?

Reply:
They probably could with some extra setup. On macOS they require keychain entitlement, which makes them easier to run from a flutter app.